crates/sui-indexer/migrations/pg/2024-09-25-211901_drop_network_total_transactions/down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE checkpoints ADD COLUMN network_total_transactions BIGINT NOT NULL DEFAULT 0;
```

Comment:
I think i'm missing some context but the PR title says we're removing `network_total_transactions` but this migration adds this column?

Reply:
this is the down.sql so it's to revert the dropping.

Reply:
although as Ashok pointed out, the empty checkpoint can be an issue following the current approach, other alternatives might change the migration here

Reply:
re adding back NOT NULL with a fake value <> adding back NULL-able column, I thought it twice and feel it's prob safer to have a slightly relaxed reverse migration by adding back NULL-able column to avoid mixing fake values and real populated values.